packages/main/src/Tokenizer.ts

```diff
@@ -57,71 +67,14 @@ import ValueStateMessageCss from "./generated/themes/ValueStateMessage.css.js";
 // reuse suggestions focus styling for NMore popup
 import SuggestionsCss from "./generated/themes/Suggestions.css.js";
 
-/**
- * @public
- */
-const metadata = {
-	tag: "ui5-tokenizer",
-	languageAware: true,
-	managedSlots: true,
-	slots: /** @lends sap.ui.webc.main.Tokenizer.prototype */ {
-		"default": {
-			propertyName: "tokens",
-			type: HTMLElement,
-			individualSlots: true,
-		},
-		"valueStateMessage": {
-			propertyName: "valueStateMessage",
-			type: HTMLElement,
-		},
-	},
-	properties: /** @lends sap.ui.webc.main.Tokenizer.prototype */ {
-		showMore: { type: Boolean },
-
-		disabled: { type: Boolean },
-
-		/**
-		 * Indicates if the tokenizer should show all tokens or n more label instead
-		 *
-		 * @private
-		 */
-		expanded: { type: Boolean },
-
-		morePopoverOpener: { type: Object },
-
-		popoverMinWidth: {
-			type: Integer,
-		},
-
-		/**
-		 * Indicates the value state of the related input component.
-		 *
-		 * @type {sap.ui.webc.base.types.ValueState}
-		 * @defaultvalue "None"
-		 * @private
-		 */
-		valueState: {
-			type: ValueState,
-			defaultValue: ValueState.None,
-		},
-
-		_nMoreCount: { type: Integer },
-	},
-	events: /** @lends sap.ui.webc.main.Tokenizer.prototype */ {
-		"token-delete": {
-			detail: {
-				ref: { type: HTMLElement },
-			},
-		},
-
-		"show-more-items-press": {
-			detail: {
-				ref: { type: HTMLElement },
-			},
-		},
-	},
-};
+type TokenizerTokenDeleteEventDetails = {
```

Comment:
I see the event detail type is not exported, probably this event detail will be need for the MultiInput/MultiCombobox, but it can be exported then

Reply:
Done